core/eolearn/core/visualizations.py

```diff
@@ -0,0 +1,547 @@
+    dataframe = xr.DataArray(data=data,
+                             coords=coordinates,
+                             dims=dimensions,
+                             attrs={'crs': str(bbox.crs),
+                                    'feature_type': feature_type,
+                                    'feature_name': feature_name},
+                             name=feature_name.replace('-', '_'))
+    if remove_depth and dataframe.values.shape[-1] == 1:
+        dataframe = dataframe.squeeze()
+        dataframe = dataframe.drop(feature_name + '_dim')
+
+    return dataframe
+
+
+def eopatch_to_dataset(eopatch, remove_depth=True):
+    """
+    Converts eopatch to xarray Dataset
+
+    :param eopatch: eopathc
+    :type eopatch: EOPatch
+    :param remove_depth: removes last dimension if it is one
+    :type remove_depth: bool
+    :return: dataset
+    :rtype: xarray Dataset
+    """
+    dataset = xr.Dataset()
+    for feature in eopatch.get_feature_list():
+        if not isinstance(feature, tuple):
+            continue
+        feature_type = feature[0]
+        feature_name = feature[1]
+        if feature_type not in (FeatureType.VECTOR, FeatureType.VECTOR_TIMELESS, FeatureType.META_INFO):
+            dataframe = array_to_dataframe(eopatch, (feature_type, feature_name), remove_depth)
+            dataset[feature_name] = dataframe
+
+    return dataset
+
+
+def new_coordinates(data, crs, new_crs):
+    """ Returns coordinates for xarray DataArray/Dataset in new crs.
+
+    :param data: data for converting coordinates for
+    :type data: xarray.DataArray or xarray.Dataset
+    :param crs: old crs
+    :type crs: BBox.crs
+    :param new_crs: new crs
+    :type new_crs: BBox.crs
+    :return: new x and y coordinates
+    :rtype: (float, float)
+    """
+    x_values = data.coords['x'].values
+    y_values = data.coords['y'].values
+    bbox = (x_values[0], y_values[0], x_values[-1], y_values[-1])
+    bbox = BBox(bbox, crs=crs)
+    bbox = bbox.transform(new_crs)
+    xmin, ymin = bbox.get_lower_left()
+    xmax, ymax = bbox.get_upper_right()
+    new_xs = np.linspace(xmin, xmax, len(x_values))
+    new_ys = np.linspace(ymin, ymax, len(y_values))
+
+    return new_xs, new_ys
+
+
+class Visualization:
+    """
+    Plot class for making visulizations.
+
+    :param eopatch: eopatch
+    :type eopatch: EOPatch
+    :param feature: feature of eopatch
+    :type feature: (FeatureType, str)
+    :param rgb: bands for creating RGB image
+    :type rgb: [int, int, int]
+    :param rgb_factor: multiplication factor for constructing rgb image
+    :type rgb_factor: float
+    :param vdims: value dimensions for plotting geopandas.GeoDataFrame
+    :type vdims: str
+    :param timestamp_column: geopandas.GeoDataFrame columns with timestamps
+    :type timestamp_column: str
+    :param geometry_column: geopandas.GeoDataFrame columns with geometry
+    :type geometry_column: geometry
+    """
```

Comment:
`pixel` and `mask` parameters are not documented.

Reply:
Added pixel and mask parameters in documentation.